classes/class-aal-settings.php

```diff
@@ -403,31 +419,31 @@ public static function select_field( $args ) {
 
 		if ( empty( $options ) || empty( $id ) || empty( $page ) )
 			return;
-		
+
 		?>
 		<select id="<?php echo esc_attr( $id ); ?>" name="<?php printf( '%s[%s]', esc_attr( $page ), esc_attr( $id ) ); ?>">
 			<?php foreach ( $options as $name => $label ) : ?>
-			<option value="<?php echo esc_attr( $name ); ?>" <?php selected( $name, (string) $value ); ?>>
-				<?php echo esc_html( $label ); ?>
-			</option>
+				<option value="<?php echo esc_attr( $name ); ?>" <?php selected( $name, (string) $value ); ?>>
```

Comment:
Just remove the space from you open the PHP tag. the `selected()` function add later..